rusk-wallet/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rand_core = "0.6"
 requestty = "0.3"
 base64 = "0.13"
 crypto = "0.3"
+toml = "0.5.8"
```

Comment:
nit: Use `major/minor`, unless strictly required - and if that is the case, put a comment documenting why you are picking also patch

Reply:
Yup, this one slipped, thanks